MC-40582: Update the install page with how to download the SUT tool via command line (#8549)
magento/devdocs: src/safe-upgrade-tool/run.md

```

{:.bs-callout-info}
The `INSTALLATION_DIR` value is the directory where your Magento instance is located.

We recommend running the following command to avoid memory limitations:

```bash
php -d memory_limit=-1 /bin/sut
```

> As well, it is recommended to use the `-m` command to run the tool against a specific module.

Reviewer: Just a minor editorial suggestion
```suggestion
We also recommend using the `-m` command to run the tool against a specific module.
```

Author: done
